src/pushOrReplace/pushOrReplace.ts

```diff
@@ -0,0 +1,50 @@
+import { push } from '../push/push';
+
+/**
+ * @module array=>array
+ */
+/**
+ * Push an element into an array, or replace an existing one if it matchs one.
```

Comment:
Document and test the case: `pushOrReplace({ match: x => x%2 === 0, replacement: 2 })([4, 6, 8]) === [2, 2, 2]`

Reply:
Done